### ISSUE TITLE
Compliance Data update automation is failing

### ISSUE THREAD

**Issue by kmcquade** [salesforce/aws-allowlister#88]
See the failed GitHub actions: https://github.com/salesforce/aws-allowlister/actions/runs/1032122283

**Comment by jdyke** [comment]
It looks like it is breaking on a new entry into the table `AWS Diode` from what I assume is a missing `<a href="">` block

![image](https://user-images.githubusercontent.com/19151605/126352933-10edf864-3a9f-4b06-b86b-0674b3299327.png)

**Comment by jdyke** [comment]
Here is an example working `service_name_cell = some_cells[0].contents[1]` vs not:

Working:
```
[<td>
    <b>
        <a href="/datasync/">
            AWS DataSync
        </a>
    </b>
</td>, <td>
    ✓
</td>, <td>
    ✓
</td>, <td>
    ✓
</td>, <td>
    ✓
</td>, <td>
</td>]
```

Not working:
```
[<td>
    AWS Diode
</td>, <td>
</td>, <td>
</td>, <td>
</td>, <td>
</td>, <td>
    DISA Review
</td>]
```

**Comment by kmcquade** [comment]
Interesting. I'll have to take a look at this. Thanks for debugging

**Comment by PatMyron** [comment]
think this issue should still be open:
https://github.com/salesforce/aws-allowlister/actions/workflows/update-compliance-data.yml?query=is:success
https://github.com/salesforce/aws-allowlister/issues/97#issuecomment-954426149

**Comment by kmcquade** [comment]
Fixed by #99
